nav2_msgs/action/Wait.action

```diff
@@ -0,0 +1,7 @@
+#goal definition
+int32 time
```

Comment:
Maybe use `builtin_interfaces/msg/Time`

Reply:
I'm not sure that makes sense given the use. 

We have a BT tag `Wait` that sets a duration to sleep for that particular context. If we do that, we'd be finding the current time, adding that duration to it. Then in the recovery action we'd be getting that time, and then subtracting the current time again to get a duration to sleep for. Seems like that information would just get in the way a bit

Reply:
Ups, I meant to use `builtin_interfaces/msg/Duration`

Reply:
Sure, no worries. Updated